Purchases/Purchasing/StoreKit1/ProductsFetcherSK1.swift

```diff
@@ -26,7 +26,7 @@ class ProductsFetcherSK1: NSObject {
     private var completionHandlers: [Set<String>: [Callback]] = [:]
     private let requestTimeout: DispatchTimeInterval
 
-    private static let numberOfRetries: Int = 5
+    private static let numberOfRetries: Int = 10
```

Comment:
Perhaps we should document somewhere in the class that this is going to return after whichever comes first of: 
- a success
- retries up to `numberOfRetries`
- timeout of 30 secs

I think the code is clear, but just to save someone from actually looking through it if they wonder how the class works

Reply:
Done.